db/table_cache.cc

```diff
@@ -336,10 +337,21 @@ Status TableCache::Get(const ReadOptions& options,
 
     if (auto row_handle =
             ioptions_.row_cache->Lookup(row_cache_key.GetUserKey())) {
+
+      // Cleanable routine to release the cache entry
+      Cleanable cache_entry_clean;
+      auto release_cache_entry_func = [] (void* cache_to_clean, void* cache_handle){
+          ((Cache*)cache_to_clean)->Release((Cache::Handle*) cache_handle);
+      };
       auto found_row_cache_entry = static_cast<const std::string*>(
           ioptions_.row_cache->Value(row_handle));
-      replayGetContextLog(*found_row_cache_entry, user_key, get_context);
-      ioptions_.row_cache->Release(row_handle);
+      // If it comes here value is located on the cache, input PinnableSlice (get_context.pinnable_slice_) will now
```

Comment:
Lets try again to make the comment more clear.

It says "input PinnableSlice (get_context.pinnable_slice_) will now point to"? It is pointing now or it will point after replayGetContextLog?

Would be great to clarify these points in the comment:
1. found_row_cache_entry points to the value on cache
2. cache_entry_clean (that you will rename it to value_pinner) will have the cleanup procedure for the cached entry.
3. After replayGetContextLog, get_context.pinnable_slice_ might point to the cached entry (or a copy based on that) and the value_pinner cleanups are delegated to get_context.pinnable_slice_
4. The cached entry is released after get_context.pinnable_slice_ is reset.